src/components/Article.jsx

```diff
@@ -5,29 +5,72 @@ import Card from 'react-bootstrap/Card';
 import user from '../images/user-icon.png'
 import clock from '../images/clock-icon.png'
 import comment from '../images/comment-icon.png'
-import vote from '../images/vote-icon.png'
+import downvote from '../images/downvote.png'
+import upvote from '../images/upvote.png'
 import Spinner from 'react-bootstrap/Spinner';
 import React from 'react'
 import ReactTimeAgo from 'react-time-ago'
 import CommentsList from './CommentsList';
+import { addArticleVotes } from '../api/api';
+import { subtractArticleVotes } from '../api/api';
 
 const Article = () => {
     const { article_id } = useParams()
     const [article, setArticle] = useState({})
     const [isLoading, setIsLoading] = useState(true)
+    const [articleVotes, setArticleVotes] = useState(0)
+    const [isAddDisabled, setIsAddDisabled] = useState(false);
+    const [isSubtractDisabled, setIsSubtractDisabled] = useState(false);
+    const [request, setRequest] = useState(true)
+
+    const createdAt = Date.parse(article.created_at);
 
     useEffect(() => {
         setIsLoading(true)
         getArticle(article_id)
             .then((response) => {
                 setIsLoading(false)
                 setArticle(response.data.article)
+                setArticleVotes(response.data.article.votes)
             })
             .catch((error) => {
                 console.log(error)
             })
     }, [])
 
+    const increaseVotes = (event) => {
+        return addArticleVotes(article_id)
+            .then((response) => {
+                if (response.status === 200) {
+                    setArticleVotes((currentVotes) => {
+                        return currentVotes + 1;
+                    })
+                    setIsAddDisabled(true);      
+                } else {
+                    setRequest(false)
+                }
+            })
+            .catch((error) => {
+                console.log(error)
+            })
+    }
+
+    const decreaseVotes = () => {
+        return subtractArticleVotes(article_id)
+            .then((response) => {
+                if (response.status === 200) {
+                    setArticleVotes((currentVotes) => {
+                        return currentVotes - 1;
+                    })
+                 setIsSubtractDisabled(true)
```

Comment:
Currently a user can upvote for example, then down, so it's back to normal, but then it's all disabled, but what if they still want to vote? Also it could work in a way that lets's say we start with 5 votes, then it can go up to 6, then go down to 5, then go down to 4, then back up to 5 again. So one extra vote in each direction. But this is just a suggestion. It's optional.